NewSpring/Rock
# added service note attribute value to service times data shortcode

## DESCRIPTION

This PR adds the ability to add a text note for a specific service in the service times matrix for campuses. This is useful for details that pertain to a specific campus/service.

### How do I test this PR?

- Go to General Settings > Campuses, and edit a campus.
- Scroll down to the "Service Times" matrix attribute. Click on/add a row, and add a value in the "Service Note" attribute field.
- The note you added should be visible on the locations swiper on one of the following pages, depending on the type of service that you added the note to.

https://newspring.cc/locations
https://newspring.cc/fuse/locations

- "Get Directions" button should only be visible if there is a value in the directions attribute that corresponds with the service's type.

## TODO


## REVIEW


> The purpose of PR Review is to _improve the quality of the software._
